scripts/build-docs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
```

Comment:
Nice. I wonder how much effort it would be to use travis-sphinx instead. Maybe that would simplify maintenance? But if you want to keep this instead, that would be ok for me as well.